samples/Calculator/tests/Calculator.Console.Tests/GlobalUsings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
I will remove GlobalUsings as there is only one file in project. Keep it minimal.

Reply:
Removed